### PR TITLE
Launch file for running gz_bridge

### PR DESCRIPTION
**Pull request by caguero**
# 🎉 New feature

Part of https://github.com/gazebosim/ros_gz/issues/544

## Summary

This patch allows to run the `ros_gz` bridge as an executable or as a composable node.

### How to test it?

Use the provided `ros_gz_bridge.launch.py` (modify `config_file` accordingly) to run the bridge as an executable

```
ros2 launch ros_gz_bridge ros_gz_bridge.launch.py config_file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_bridge/test/config/full.yaml
```

Verify that you see the `/ros_chatter` topic:
```
caguero@cold:~/ros_gz_ws$ ros2 topic list
/parameter_events
/ros_chatter
/rosout
```

And confirm that no container has been created:
```
caguero@cold:~/ros_gz_ws$ ros2 component list

```

Now `CTRL-C` the bridge and try the composable version:
```
 ros2 launch ros_gz_bridge ros_gz_bridge.launch.py config_file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_bridge/test/config/full.yaml use_composition:=True
```

Verify that you see the `/ros_chatter` topic:
```
caguero@cold:~/ros_gz_ws$ ros2 topic list
/parameter_events
/ros_chatter
/rosout
```

And confirm that a container has been created:
```
caguero@cold:~/ros_gz_ws$ ros2 component list
/ros_gz_container
  1  /ros_gz_bridge
```

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (ahcorde) [inline ros_gz_bridge/launch/ros_gz_bridge.launch.py]: remove first empty line

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    log_level = LaunchConfiguration('log_level')
+
+    declare_config_file_cmd = DeclareLaunchArgument(
+        'config_file', default_value='', description='YAML config file'
+    )
+
+    declare_container_name_cmd = DeclareLaunchArgument(
+        'container_name',
+        default_value='ros_gz_container',
+        description='the name of container that nodes will load in if use composition',
```
Comment (ahcorde) [inline ros_gz_bridge/launch/ros_gz_bridge.launch.py]:
```suggestion
        description='Name of container that nodes will load in if use composition',
```

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    use_respawn = LaunchConfiguration('use_respawn')
+    log_level = LaunchConfiguration('log_level')
+
+    declare_config_file_cmd = DeclareLaunchArgument(
+        'config_file', default_value='', description='YAML config file'
+    )
+
+    declare_container_name_cmd = DeclareLaunchArgument(
+        'container_name',
+        default_value='ros_gz_container',
```
Comment (azeey) [inline ros_gz_bridge/launch/ros_gz_bridge.launch.py:40]: Note: The default here should match what we name container in https://github.com/gazebosim/ros_gz/pull/528

**Review by azeey — APPROVED** [review]
LGTM

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py (hunk 3 above)
Comment (caguero) [reply ros_gz_bridge/launch/ros_gz_bridge.launch.py:40]: Updated [ff3149b](https://github.com/gazebosim/ros_gz/pull/530/commits/ff3149b9c6b4d8cfe666ee0aac37361fd23ed74c).

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py (hunk 2 above)
Comment (caguero) [reply ros_gz_bridge/launch/ros_gz_bridge.launch.py]: Updated in [ff3149b](https://github.com/gazebosim/ros_gz/pull/530/commits/ff3149b9c6b4d8cfe666ee0aac37361fd23ed74c)

File: ros_gz_bridge/launch/ros_gz_bridge.launch.py (hunk 1 above)
Comment (caguero) [reply ros_gz_bridge/launch/ros_gz_bridge.launch.py]: Removed in [ff3149b](https://github.com/gazebosim/ros_gz/pull/530/commits/ff3149b9c6b4d8cfe666ee0aac37361fd23ed74c)

**Pull request merged**